linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_silent/oval/shared.xml

```diff
@@ -18,6 +18,9 @@
                 <criterion
                 test_ref="test_pam_faillock_silent_parameter_password_auth"
                 comment="Check the silent parameter in auth section of password-auth file"/>
+                <criterion
```

Comment:
The CI fail of the Ansible remediation for this rule looks a valid fail. The Ansible remediation seems to put the "silent"  option on all 3 files at the same time (password-auth, system-auth and faillock.conf). But the OVAL requires that the option is set to ether password-auth and system-auth or failock.conf but not all at the same time. I think the Ansible code needs to be adjusted so that it wouldn't add the option to faillock.conf if it added the option to the other files.